azure-template/azure/template/template_code.py

```diff
@@ -0,0 +1,2 @@
+def template_main():
+    print("Call your main application code here")
```

Comment:
I would be cautious about using the work "application" in a template about SDK

Reply:
Fair point. I'll update to "package"